README.md

```diff
@@ -15,17 +15,18 @@ LinuxKit, a toolkit for building custom minimal, immutable Linux distributions.
 
 ## Getting Started
 
-### Build the `moby` tool
+### Build the `moby` and `linuxkit` tools
```

Comment:
I still am trying to wrap my head around it. Is the short form: `moby` is a tool for building images, and `linuxkit` is a tool for pushing and running images? So `moby == docker build` and `linuxkit == docker run / docker push`?